corpora/proverbs.dep

```diff
@@ -27,3 +28,979 @@
+"<,>"
+	"," cm @punct #4->7
+"<мойнуң>"
+	"моюн" n px2sg nom @obl #5->7
+"<менен>"
+	"менен" post @cc #6->5
+"<тартасың>"
+	"тарт" v tv aor p2 sg @root #7->0
+"<.>"
+	"." sent @punct #8->7
+
+# sent_id = proverbs:22
+# author:id = Сыргабековна&Сабыровна:465
+# text = Шашкан шайтандын иши.
+# text[eng] = Hurrying is what a shaytan does. / Hurrying is the devil's work.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Шашкан>"
+	"шаш" v iv ger_past nom @nsubj #1->3
+"<шайтандын>"
+	"шайтан" n gen @nmod:poss #2->3
+"<иши>"
+	"иш" n px3sp nom @root #3->0
+		"э" cop aor p3 sg @cop #4->3
+"<.>"
+	"." sent @punct #5->3
+
+# sent_id = proverbs:23
+# author:id = Сыргабековна&Сабыровна:313
+# text = Куру сөз курсак тойгузбайт.
+# text[eng] = A dry word does not fill a belly.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Куру>"
+	"куру" adj @amod #1->2
+"<сөз>"
+	"сөз" n nom @nsubj #2->4
+"<курсак>"
+	"курсак" n nom @obj #3->3
+"<тойгузбайт>"
+	"той" v iv caus neg aor p3 sg @root #4->0
+"<.>"
+	"." sent @punct #5->4
+
+# sent_id = proverbs:24
+# author:id = Сыргабековна&Сабыровна:333
+# text = Мурунку көч кайда барса, кийинки көч да ошол жакка барат.
+# text[eng] = Where the former migration goes, the next migration will go there too.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Мурунку>"
+	"мурун" adv attr @amod #1->2
+"<көч>"
+	"көч" n nom @csubj #2->4
+"<кайда>"
+	"кайда" adv itg @advmod #3->4
+"<барса>"
+	"бар" v iv gna_cond p3 sg @advcl #4->11
+"<,>"
+	"," cm @punct #5->11
+"<кийинки>"
+	"кийин" adv attr @amod #6->7
+"<көч да>"
+	"көч" n nom @nsubj #7->11
+		"да" postadv @advmod #8->11
+"<ошол>"
+	"ошол" det dem @det #9->10
+"<жакка>"
+	"жак" n dat @obl #10->11
+"<барат>"
+	"бар" v iv aor p3 sg @root #11->0
+"<.>"
+	"." sent @punct #12->11
+
+# sent_id = proverbs:25
+# author:id = Сыргабековна&Сабыровна:499
+# text = Эшекке төшөк жарашпайт.
+# text[eng] = A sleeping mat is not suitable for a donkey.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Эшекке>"
+	"эшек" n dat @obl #1->3
+"<төшөк>"
+	"төшөк" n nom @nsubj #2->3
+"<жарашпайт>"
+	"жара" v tv coop neg aor p3 sg @root #3->0
+"<.>"
+	"." sent @punct #4->3
+
+# sent_id = proverbs:26
+# author:id = Сыргабековна&Сабыровна:275
+# text = Кетмен чаппай - нан жок, чеке терсиз - мал жок.
+# text[eng] = Without the hoe striking there's no bread, without a sweaty forehead there's no livestock.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Кетмен>"
+	"кетмен" n nom @csubj #1->2
+"<чаппай>"
+	"чап" v tv neg gna_perf @advcl #2->5
+"<->"
+	"-" guio @punct #3->5
+"<нан>"
+	"нан" n nom @nsubj #4->5
+"<жок>"
+	"жок" adj @parataxis #5->13
+		"э" cop aor p3 sg @cop #6->5
+"<,>"
+	"," cm @punct #7->13
+"<чеке>"
+	"чеке" n nom @csubj #8->9
+"<терсиз>"
+	"тер" n @advcl #9->13
+		"сыз" post @case #10->9
+"<->"
+	"-" guio @punct #11->13
+"<мал>"
+	"мал" n nom @nsubj #12->13
+"<жок>"
+	"жок" adj @root #13->0
+		"э" cop aor p3 sg @cop #14->13
+"<.>"
+	"." sent @punct #15->13
```

Comment:
The structure of this looks really good.

Reply:
Thanks!